main.go

```diff
@@ -135,78 +224,76 @@ func wrapMain() error {
 
 	e := os.Environ()
 	e = append(e, fmt.Sprintf("GOOGLE_APPLICATION_CREDENTIALS=%s", keyPath))
-	runner := NewEnviron(workspace.Path, e, os.Stdout, os.Stderr)
+	wd, err := os.Getwd()
```

Comment:
using `os.Getwd()` instead of `workspace.Path` seems to work in my limited testing, but I'm new to golang and I would be very surprised if this is the correct/best practice way to handle this, can anyone more fluent in golang please review?